healthcheck/healthchecksTest.yml

```diff
@@ -0,0 +1,17 @@
+name: rhobot healthcheck test
+distribution:
+- stefan.fox@cfpb.gov
```

Comment:
True, we should use dummy emails here, just to be safe.  Normally these YAML files will be internal only.  @frank0631 can you make the change and purge the git repo?